File: guanabara-java/src/com/danillo/algoritmos/lista01/Exercicio08.java
package com.danillo.algoritmos.lista01;

import java.util.Scanner;

public class Exercicio08 {
    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);

        System.out.println("Digite uma distância em metros: ");
        float m = entrada.nextFloat();

        float km = m / 1000;
        float hm = m / 100;
        float dam = m/ 10;
        float dm = m * 10;
        float cm = m * 100;
        float mm = m * 1000;

        System.out.println("A distância de " +m+ "m corresponde a: \n"
                            +km+ "km\n" +hm+ "hm\n" +dam+ "dam\n" +dm+ "dm\n" +cm+ "cm\n" +mm+ "mm\n");
    }
}

File: guanabara-c/lista01/exercicio08.c
#include <stdio.h>

int main(){
    float m, km, hm, dam, dm, cm, mm;

    printf("Insira uma distância em metros: ");
    scanf("%f", &m);

    km = m / 1000;
    hm = m / 100;
    dam = m / 10;
    dm = m * 10;
    cm = m * 100;
    mm = m * 1000;
    printf("A distância de %.2f metros corresponde a:\n%.2fkm\n%.2fhm\n%.2fdam\n%.2fdm\n%.2fcm\n%.2fmm\n", m, km, hm, dam, dm, cm, mm);

    return 1;
}
